### ISSUE TITLE
Clickable elements are too small for WCAG AA and AAA compliance

### ISSUE THREAD

**Issue by brittnylapierre** [ProjectMirador/mirador#4232]
How to move to the next page and enlarge the image
21px size
All icon buttons should be at least 24px for clickable elements for AA - 44px for AAA
At different breakpoints we will want 44px or 48px

Accessibility Impacts:

Screenshots:
